Source/OxyPlot/PlotModel/ITransposablePlotElement.cs

```diff
@@ -9,21 +9,10 @@
 
 namespace OxyPlot
 {
-    using OxyPlot.Axes;
-
     /// <summary>
     /// The TransposablePlotElement interface.
     /// </summary>
-    public interface ITransposablePlotElement : IPlotElement
+    public interface ITransposablePlotElement : IXyAxisPlotElement
```

Comment:
Given we are keeping this, we can probably re-fit the example library `PlotModelUtilites` without the hashsets.

Reply:
For the annotations yes, but for `Series` I think for now we still need to hardcode `CandleStickAndVolumeSeries` and `OldCandleStickSeries`.